cmd/hope/deploy.go

```diff
@@ -19,15 +19,26 @@ import (
 	"github.com/Eagerod/hope/pkg/kubeutil"
 )
 
+var deployCmdTagSlice *[]string
+
+func initDeployCmdFlags() {
+	deployCmdTagSlice = deployCmd.Flags().StringArrayP("tag", "t", []string{}, "deploy resources with this tag")
```

Comment:
Could also have a flag `--tags` that accepts a comma separated list, but then can't have commas in tags.